### ISSUE TITLE
proxmark3> 'emv scan -w'  always fails

### ISSUE THREAD

**Issue by pwpiwi** [Proxmark/proxmark3#768]
```
proxmark3> emv scan -w test.json
ERROR: Do not use contact interface. Exit.
proxmark3>
```
Is this work in progress? Having a `-w` option just to create an error?

**Comment by merlokk** [comment]
yes, it not completed yet. `scan` not the same for contact interface
Now im working on VSDC transaction. and after that ill adapt it

**Comment by pwpiwi** [comment]
fixed with PR #789
